docs/source/tutorials-basic/driving.rst

```diff
@@ -0,0 +1,201 @@
+.. _tutorials-basic-driving-chapter:
+
+**************************
+Driving Around
+**************************
+
+Driving the GoPiGo3 around is only a matter of choosing the right methods from the :py:class:`~easygopigo3.EasyGoPiGo3` class.
+In all of our examples, the :py:mod:`~easygopigo3` is required and needs to be imported and a :py:class:`~easygopigo3.EasyGoPiGo3` object
+has to be instantiated this way:
```

Comment:
this sounds like it's the only way to instantiate it.  Just rephrase to something less limiting, like "can be instantiated this way"